app/src/assets/placeholder_round.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="1920" height="1080" viewBox="0 0 1920 1080" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this the right SVG file? If so what exactly is it for? It seems nearly identical to `placeholder_grant.svg` 
![image](https://user-images.githubusercontent.com/17163988/132913339-e3f00387-441a-48eb-8932-411bb69f4984.png)